.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
I guess we could improve the [documentation](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types#mode) for that mode.
We ship the same code to npm no matter what the user environment is.